README.md

```diff
@@ -185,6 +185,14 @@ Note that although the annotation containing router configuration for each of th
 | deis-router          | workerProcesses  | `string`   | `auto` (number of CPU cores) | Number of worker processes to start. |
 | deis-router          | workerConnections | `integer` | `768`         | Maximum number of simultaneous connections that can be opened by a worker process. |
 | deis-router          | defaultTimeout   | `integer`  | 1300          | Default timeout value in seconds.  Should be greater than the front-facing load balancer's timeout value. |
+| deis-router          | useGzip          | `boolean`  | `true`        | Whether to use gzip compression. |
```

Comment:
Ok... this can work... this in the annotations:

```
metadata:
  annotations:
    deis.io/routerConfig: |
      {
        "domain": "example.com",
        "useProxyProtocol": false,
        "gzipConfig": null
      }
```

And this simple change in the template:

```
{{ if $routerConfig.GzipConfig }}{{ $gzipConfig := $routerConfig.GzipConfig }}gzip on;
```

I guess that with the proper docs, this is a reasonable means of wiping out the default configuration to completely disable gzip.

I'm going to make that change.